store.go

```diff
@@ -1455,6 +1462,42 @@ func (s *store) ImageBigDataSize(id, key string) (int64, error) {
 	return -1, ErrSizeUnknown
 }
 
+func (s *store) MountTempFromSource(id, source string) (string, error) {
+	rcstore, err := s.ContainerStore()
+	if err != nil {
+		return "", err
+	}
+	rcstore.RLock()
+	defer rcstore.Unlock()
```

Comment:
Missing a `Modified()` check and `Load()` here.

Reply:
Added.